### PR TITLE
Add jaeger as datasource to Grafana

### PR DESCRIPTION
**Pull request by cedricziel**
This adds jaeger as a datasource for trace data to Grafana when provisioning.

### CODE REVIEW

**Comment by mviitane** [comment]
Looks good! I think the grafana bullet point in the main README could also be updated to mention traces.

File: src/grafana/provisioning/datasources/default.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 apiVersion: 1
 
 datasources:
-  - name: webstore
+  - name: webstore-metrics
```
Comment (reyang) [inline src/grafana/provisioning/datasources/default.yaml:4]: Maybe:

```suggestion
  - name: Prometheus
```

File: src/grafana/provisioning/datasources/jaeger.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
+
+datasources:
+  - name: webstore-traces
```
Comment (reyang) [inline src/grafana/provisioning/datasources/jaeger.yaml:4]: Maybe:

```suggestion
  - name: Jaeger
```

**Review by reyang — APPROVED** [review]

**Comment by julianocosta89** [comment]
Any reason why do we need that in this demo?

**Comment by cedricziel** [comment]
@julianocosta89 it's an out of the box feature of Grafana. - It enables viewing traces and metrics side by side and stays in the OpenSource track.

Jaeger is still the main store for the trace data. - Hence I dont think there is an issue adding this. It's lesser a thing we need in the demo, but we get it for free - why not enable it?

**Comment by cedricziel** [comment]
Alright :) 

Sure thing - Grafana can provision dashboards from file as well - makes sense to add a set of curated dashboards to the demo.

**Comment by cedricziel** [comment]
I would consider this one finished for now - dashboards would be a separate PR :)

**Review by fatsheep9146 — APPROVED** [review]

**Review by puckpuck — APPROVED** [review]

**Pull request merged**